src/bundle/Resources/public/js/scripts/fieldType/ezdatetime.js

```diff
@@ -63,7 +63,7 @@
     const datetimeConfig = {
         enableTime: true,
         time_24hr: true,
-        formatDate: (date) => new Date(date).toLocaleString(),
+        formatDate: (date) => eZ.helpers.timezone.formatFullDateTime(date),
```

Comment:
This one will be converted but other in flatpickr are not. Intentional?

Reply:
On the second thought, all should be converted to user timezone, this is displayed date.